spec/controllers/idv/by_mail/enter_code_rate_limited_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Idv::ByMail::EnterCodeRateLimitedController do
+  let(:user) { build_stubbed(:user, :fully_registered) }
+  let(:gpo_verification_pending_profile) { true }
+
+  before do
+    allow(user).to receive(:gpo_verification_pending_profile?).
+      and_return(gpo_verification_pending_profile)
```

Comment:
Is there a reason to stub this rather than building a user with a pending profile? Overall, we want our specs to use real data.